### PR TITLE
Initial setup & homepage content

### PR DESCRIPTION
**Pull request by michaelti**
Hey team! Did a few things in this PR:
- Added Sass
- Removed Vite boilerplate
- Added placeholder text to the homepage

### CODE REVIEW

**Review by mohamedoa — COMMENTED** [review]
Looks great !

**Comment by dnaHunter** [comment]
See no issues with this.

**Review by stephaniewaterson — COMMENTED** [review]
looks great!

File: src/App.jsx [hunk 1]
```diff
@@ -1,35 +1,14 @@
...
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
+      <h1>DogScience 🐶</h1>
+      <p>
+        Here at DogScience Labs, we are dedicated to unlocking the mysteries of canine biology, behavior, and well-being. Our state-of-the-art research facility is at the forefront of innovation, blending cutting-edge technology with a passion for improving the lives of dogs everywhere.
```
Comment (temiadenuga) [inline src/App.jsx:8]: Hey Michael. Could you please add "DogScience Labs" in a span since we need some styling in there according to the mockup?

Comment (michaelti) [reply src/App.jsx:8]: Sure, will do!

**Comment by michaelti** [comment]
@temiadenuga I updated the dog span. Can you have another look?

**Comment by temiadenuga** [comment]
Fabulous! 👍

**Pull request merged**